src/js/jsx/shared/ColorPicker.jsx

```diff
@@ -949,8 +946,6 @@ define(function (require, exports, module) {
                         currentRgbaColor.b !== nextRgbaColor.b) {
                         this._changeColor(nextRgbaColor);
                     }
-
-                    this.props.onChange(nextRgbaColor);
```

Comment:
I think the only callers of this function are changing only the opaque color or the alpha. When the text input changes, it calls `onChange` already, so it isn't needed here. But, also, see my comment above about how this is getting kind of gross... I agree that it's confusing at best.